### PR TITLE
docs: add new page helm

### PR DESCRIPTION
**Pull request by edeediong**

### CODE REVIEW

File: docs/configuration/cellsection.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+To do this, you first need to enable “Sticky scroll” in your notebook. Follow the steps below to enable sticky scroll:
+
+- Click on the setting icon at the bottom left of your VS Code editor.
+- From the provided options, select “Settings”
+
+
+![VS Code](../../static/img/guide-page/runme-vscode-setting.png)
+
+- In the search bar, enter ‘notebook sticky.’ This will direct you to the sticky scroll configuration page in the notebook.
+- Now enable Stick Scroll by clicking on the “Enabled” checkbox.
```
Comment (joseeantonior) [inline docs/configuration/cellsection.md:21]: We've been calling it "sticky scroll" but it mentions "Stick Scroll" here. We should standardize the use of the term to either "the Stick Scroll feature" or "sticky scroll".

File: docs/guide/helm.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+# How To Integrate Helm With Runme
+
+If you are performing a task repeatedly, it is best to automate it; this is a core principle of DevOps.
+
+With Runme, you can automate your task by creating an interactive notebook where you can run your commands, perform deployment and carry out the instructions outlined in your documentation.
+
+This guide demonstrates how to integrate Helm with Runme to automate processes and reduce repetitive tasks.
```
[the remark this answers is not in the document]
Comment (joseeantonior) [reply docs/guide/helm.md:7]: There, it's actually a package manager, and not an automation tool. We should definitely be explicit.

**Review by sourishkrout — CHANGES_REQUESTED** [review]
Please see comment about notebook, otherwise 👍 .

File: docs/configuration/cellsection.md (hunk 1 above)
Comment (edeediong) [reply docs/configuration/cellsection.md]: We are going to stick with "sticky scroll".

**Pull request merged**